src/libsync/networkjobs.cpp

```diff
@@ -346,7 +346,7 @@ void LsColJob::start()
         qCWarning(lcLsColJob) << "Propfind with no properties!";
     }
     QByteArray propStr;
-    foreach (const QByteArray &prop, properties) {
+    for (const QByteArray &prop : properties) {
```

Comment:
This is outside of the scope of your changes, but, why are we creating a copy of the member `_properties` higher up in this method? It doesn't look like we need to do this

We can delete the variable and just depend on a `std::as_const(_properties)` instead here